src/bindings/js/node/src/tensor.cpp

```diff
@@ -191,3 +192,42 @@ Napi::Value TensorWrap::is_continuous(const Napi::CallbackInfo& info) {
     }
     return Napi::Boolean::New(env, _tensor.is_continuous());
 }
+
+void TensorWrap::set_shape(const Napi::CallbackInfo& info) {
+    Napi::Env env = info.Env();
+
+    if (!info[0].IsArray()) {
```

Comment:
Please validate values using [this approach](https://github.com/openvinotoolkit/openvino/pull/24921). See also [this PR](https://github.com/openvinotoolkit/openvino/pull/27148)